src/org-manager/components/organisation-details-info/models/organisation-details.ts

```diff
@@ -0,0 +1,18 @@
+export enum RequestType {
+  APPROVE_REQUEST = 'Approve it',
+  REJECT_REQUEST = 'Reject it',
+  REVIEW_REQUEST = 'Place registration under review pending further investigation',
+}
+export interface ErrorMessage {
+  title: string;
+  description: string;
+  fieldId?: string;
+}
+export interface DisplayedRequest {
+  request: RequestType;
+  checked: boolean;
+}
+export enum RequestErrors {
+  NO_SELECTION = 'Select what would you like to do with this registration',
```

Comment:
Where has this error text come from?

"Select what would you like.... "sounds a bit tongue-twisting, should it not be "Select what you would like..."

Reply:
It comes from the requirements from the ticket (scenario :4)